testpilot/frontend/static-src/app/views/experiment-tour-dialog-view.js

```diff
@@ -81,10 +82,22 @@ export default BaseView.extend({
 
   cancel() {
     this.animateRemove();
+
+    app.sendToGA('event', {
+      eventCategory: 'ExperimentDetailsPage Tour Modal Interactions',
```

Comment:
I don't think we have to change the `eventCategory` from [other interactions on the experiment details page](https://github.com/mozilla/testpilot/blob/master/testpilot/frontend/static-src/app/views/experiment-page.js#L233).

"ExperimentDetailsPage Interactions" will be okay for all of these.